### PR TITLE
minor fix to breakpoint logic https://github.com/jeffeb3/sandify/issues/276

### PR DESCRIPTION
**Pull request by aaronse**
minor fix to breakpoint logic https://github.com/jeffeb3/sandify/issues/276

### CODE REVIEW

File: src/features/app/App.scss [hunk 1]
```diff
@@ -39,7 +39,7 @@
   width: 550px;
   padding-top: 0.5rem;
 
-  @media(max-width: 992px) {
+  @media not (min-width: 992px) {
```
Comment (bobnik) [inline src/features/app/App.scss:42]: Instead of using "not" here, just change min-width to 991px

Comment (aaronse) [reply src/features/app/App.scss:42]: not seemed like more robust way to avoid this issue in the first place?

Comment (bobnik) [reply src/features/app/App.scss:42]: Disagree. Breakpoint is exactly at 992px. No need for a more general case, and I do not find "not min-width" to be intuitive. You're welcome to make the change, or I can fix independently.

**Review by aaronse — COMMENTED** [review]
Applied feedback from @bobnik

**Pull request merged**